### PR TITLE
cleanerChannels

### PR DESCRIPTION
**Pull request by jsh3d**
Removes channel icons to give discord a more simple look. Will add customization and a readme.md soon, this is just to see if this will get added.

### CODE REVIEW

**Comment by jsh3d** [comment]
yo can someone close this and tell me why or add it

File: src/plugins/cleanerChannels/index.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            replace: "iconBlowhorn:{}",
+        },
+        {
+            find: /iconRules:\{[^}]*icon:/,
+            replacement: () => "",
+            match: /iconRules:\{[^}]*icon:"rules"/,
+            replace: "iconRules:{}",
+        },
+    ],
+    start() {
```
Comment (Sqaaakoi) [inline src/plugins/cleanerChannels/index.ts:28]: ?????? so useless these aren't mandatory to have at all

Comment (Sqaaakoi) [reply src/plugins/cleanerChannels/index.ts:28]: also yes like others said just use CSS

**Comment by jsh3d** [comment]
> if you want to close a pr you can do it yourself. just click here:
> 
> 
> 
> this likely won't be merged since it can be done using a CSS snippet, which doesn't require any plugin. general "policy" with vencord plugins are that they shouldn't be made if you can do it with CSS

ik i just need a response from the admins or janitors

**Comment by MrDiamondDog** [comment]
> ik i just need a response from the admins or janitors

If bet they're going to say the same thing as we did

**Comment by Vendicated** [comment]
as others have stated, this can easily be achieved with css, so a plugin is not necessary for this

thank you for your contribution nevertheless!

**Comment by jsh3d** [comment]
👍

**Pull request closed without merging**